lua/nordbuddy.lua

```diff
@@ -1,573 +1,80 @@
-    }
-    local texts = {
-        'TSText', 'TSStrong', 'TSEmphais', 'TSUnderline', 'TSStrike', 'TSTitle',
-        'TSLiteral', -- TS
-        'Text', -- VL
-        'manTitle' -- man
-    }
-    local uris = {
-        'TSURI' -- TS
-    }
-    local types = {
-        'TSType', 'TSTypeBuildin', -- TS
-        'Type', 'StorageClass', 'Structure', 'Typedef', -- VL
-        'cType', 'cStorageClass', 'cStructure', 'cppType', 'cppStorageClass',
-        'cppStructure', -- C/C++
-        'pythonclass', -- python
-        'vimlet' -- vim
-    }
-    local variables = {
-        'TSVariable', -- TS
-        'Identifier', -- VL
-        'pythonselfarg', -- python
-        'rustIdentifier', -- rust
-        'vimmapmodkey', 'vimnotation', 'vimfuncvar', 'vimvar', -- vim
-        'shDerefSimple', 'shDerefVar' -- sh
-    }
-    local builtins = { -- such as self. in python
-        'TSConstBuiltin', 'TSFuncBuiltin', 'TSVariableBuiltin', -- TS
-        'pythonbuiltin', 'pythonself', -- python
-        'vimmap' -- vim
-    }
-    local notes = {
-        'TSNote', -- TS
-        'Todo', -- VL
-        'cTODO' -- C/C++
-    }
-    local warnings = {
-        'TSWarning', -- TS
-        'Debug' -- VL
-    }
-    local dangers = {
-        'TSDanger' -- TS
-    }
-    local errors = {
-        'TSError', -- TS
-        'Error', -- VL
-        'rustCommentLineDocError', 'rustCommentBlockDocError' -- rust
-    }
-    local groups = {
-        {attributes, c.nord9}, --
-        {numbers, c.nord15}, --
-        {comments, c.nord3:light()}, --
-        {constructors, c.nord4, cno, i}, -- in C++ variable->constructors() \\ TS docs unclear
-        {conditionals, c.nord10, cno, i}, --
-        {constants, c.nord4}, --
-        {defines, c.nord10}, --
-        {exceptions, c.nord9}, --
-        {fields, c.nord9, cno}, --
-        {functions, c.nord4, cno, i}, --
-        {includes, c.nord10}, --
-        {keywords, c.nord9, cno, sno}, --
-        {labels, c.nord10, cno, i}, --
-        {namespaces, c.nord4, cno, i}, --
-        {nones, c.nord4}, --
-        {operators, c.nord9}, --
-        {parameters, c.nord4, cno, i}, --
-        {properties, c.nord9}, --
-        {punctuations, c.nord9}, --
-        {repeats, c.nord9, cno, i}, --
-        {strings, c.nord14}, --
-        {strings_specials, c.nord13}, --
-        {symbols, c.nord13}, --
-        {texts, c.nord4}, --
-        {uris, c.nord15}, --
-        {types, c.nord7}, --
-        {variables, c.nord4}, --
-        {builtins, c.nord9, cno, i}, --
-        {notes, c.nord12}, --
-        {warnings, c.nord13}, --
-        {dangers, c.nord13}, --
-        {errors, c.nord11} --
-
-    }
-
-    local highlights = {}
-
-    -- Apply grouping to each color group
-    for _, group in ipairs(groups) do
-        highlights = merge({
-            highlights, --
-            highlight_to_groups({group[2], group[3], group[4]})(group[1]) --
-        })
+function initializeColors()
```

Comment:
From what I've seen `snake_case` seems more common in lua functions.
```suggestion
function initialize_colors()
```

Reply:
https://github.com/luarocks/lua-style-guide#variable-names

Reply:
Indeed. Fixed :blush: